modules/visualization/dcv-eks/modulestack.template

```diff
@@ -0,0 +1,26 @@
+AWSTemplateFormatVersion: 2010-09-09
```

Comment:
we can remove this template if not needed

Reply:
Correct. Sorry forgot to remove these two files. Removing for this and next comment.